pkgs/applications/editors/vscode/extensions/ms-vscode.cpptools/default.nix

```
@@ -97,10 +101,15 @@ vscode-utils.buildVscodeMarketplaceExtension {
 
       # Patching binaries
       chmod +x bin/cpptools bin/cpptools-srv bin/cpptools-wordexp debugAdapters/bin/OpenDebugAD7
+    ''
+    + lib.optionalString stdenv.hostPlatform.isLinux ''
       patchelf --replace-needed liblttng-ust.so.0 liblttng-ust.so.1 ./debugAdapters/bin/libcoreclrtraceptprovider.so
     ''
     + lib.optionalString isx86Linux ''
       chmod +x bin/libc.so
+    ''
+    + lib.optionalString isDarwin ''
```

```suggestion
    + lib.optionalString stdenv.hostPlatform.isDarwin ''
```

The `isDarwin` variable only used once and can be removed.

---

It's actually used twice, the second time in:
> dontAutoPatchelf = isx86Linux || isDarwin;

Should I inline it in both places or keep it as is?